teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Ah, I see your f-strings here :)</p><p> </p><p>Could you please add an <code>else</code> to your if-elif clause?</p><pre class="ql-syntax" spellcheck="false">else:
	raise ValueError(f"Split method {split_method} undefined; use random or loto.")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>